### ISSUE TITLE
Feature Request: Generate file like: *.sha512 *.sha256 *.sha-1, etc..

### ISSUE THREAD

**Issue by lexterror** [tedsmith/quickhash#63]
Hi, I was wondering whether in a future release you could include the following features:

1: Generating files with the hash included and extension like: *.sha512 *.sha256 *.sha-1, etc..

2: The possibility to check a file or folder against the the generated files above from the gui.

Thank you!

**Comment by tedsmith** [comment]
So do you mean if the file was called MyFile.doc, you want to generate output files called Myfile.doc.sha512, and in that file, you want it to hold the hash value, or the original file content?

**Comment by lexterror** [comment]
Yes! and also being able to verify the file/folder within the gui

**Comment by lexterror** [comment]
*hash value

**Comment by okunova** [comment]
Hello could you include the ed2k algorithm to hashear
